javascript/javascript3/week2/main.js

```diff
@@ -0,0 +1,151 @@
+// Movies exercise
+// This exercise is repetition of array functions. you dont have to use chaining or anything fancy. Just fetch the movies and use the correct array function to get the following movies:
+
+// Fetch movies from this api:
+
+// Create an array of bad movies
+// Creat an array of bad movies since year 2000
+
+let result = fetch(
+  "https://gist.githubusercontent.com/pankaj28843/08f397fcea7c760a99206bcb0ae8d0a4/raw/02d8bc9ec9a73e463b13c44df77a87255def5ab9/movies.json"
+).then((response) =>
+  response
+    .json()
+    .then((arrOfAllmovies) => {
+      const badMovie = 4;
+      const arrayOfBadMovies = arrOfAllmovies.filter(
+        (movie) => movie.rating < badMovie
+      );
+      console.log(arrayOfBadMovies);
+      return arrayOfBadMovies;
+    })
+    .then((arrayOfBadMovies) => {
```

Comment:
you don't need to one more .then her you can filter the arrayOfBadMovies to get arrayOfBadMoviesSince2000

Reply:
done